feconf.py

```diff
@@ -673,7 +672,7 @@ def get_empty_ratings():
 FETCH_SKILLS_URL_PREFIX = '/fetch_skills'
 FLAG_EXPLORATION_URL_PREFIX = '/flagexplorationhandler'
 FRACTIONS_LANDING_PAGE_URL = '/fractions'
-TOPIC_LANDING_PAGE_URL = '/learn/<subject>/<topic>'
+TOPIC_LANDING_PAGE_URL = '/<topic>'
```

Comment:
I think this needs to be reworked. None of the other URL constants here have "< ...>" bits so please do whatever changes are needed to achieve consistency. We might not need this constant at all.

Reply:
Done.